src/SumoCoders/FrameworkCoreBundle/Service/JsData.php

```diff
@@ -49,6 +48,8 @@ protected function handleRequestStack()
      */
     public function __toString()
     {
+        $this->handleRequestStack();
```

Comment:
Ik vind het onlogisch dat `__toString()` side effects heeft...

Also, aangezien `$this->requestStack` via een setter kan gezet worden moet je in `handleRequestStack()` toch even checken of het niet `null` is vooraleer je er methods op gaat roepen. (is dit geen teken dat er eigenlijk constructor injection moet zijn?)